### PR TITLE
Module2 task2

### PR DESCRIPTION
**Pull request by Nastya-Korsak**
Task 1 and Task 2

### CODE REVIEW

File: Module_2/Task_1/HelloLibrary/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace HelloLibrary
+{
+    public class HelloClass
```
Comment (alexander-trishin) [inline Module_2/Task_1/HelloLibrary/Class1.cs:5]: Please, try to use meaningful names. Filename and class name should be same.

I would recommended to read this great book “Clean Code: A Handbook of Agile Software Craftsmanship” by Robert C. Martin. Reading this book will help you to write better and cleaner code.

File: Module_2/Task_1/HelloLibrary/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    {
+        public static string HelloMethod(string name)
+        {
+            var time = DateTime.Now.ToString("HH:mm:ss");
```
Comment (alexander-trishin) [inline Module_2/Task_1/HelloLibrary/Class1.cs:9]: JFYI: There is also `DateTime.Now.TimeOfDay` which will produce the time in the format `HH:mm:ss. ffffff`. This format was also suitable for the task.

File: Module_2/Task_1/HelloLibrary/Class1.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+            return $"{time} Hello {name}!";
```
Comment (alexander-trishin) [inline Module_2/Task_1/HelloLibrary/Class1.cs:11]: Case of `null` and `string.Empty` is not handled.

File: Module_2/Task_1/WinForms/Form1.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Windows.Forms;
```
Comment (alexander-trishin) [inline Module_2/Task_1/WinForms/Form1.cs:2]: `WinForms` is very outdated technology. I would recommend paying attention to the `WPF` 😅

File: Module_2/Task_1/ConsoleApp/Program.cs [hunk 5]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
```
Comment (alexander-trishin) [inline Module_2/Task_1/ConsoleApp/Program.cs:3]: 👍 for finding this library.

Why reinvent the wheel when you already have a solution that allows you to build a convenient console interface? 😅 Good job.

**Review by alexander-trishin — APPROVED** [review]
Hey Nastya 😃 

Not bad for the first task. Overall, nothing critical. But I would recommend you to fix the code, according to my comments.

File: Module_2/Task_1/HelloLibrary/Class1.cs (hunk 1 above)
Comment (Nastya-Korsak) [reply Module_2/Task_1/HelloLibrary/Class1.cs]: Changed class name and method name

File: Module_2/Task_1/HelloLibrary/Class1.cs (hunk 3 above)
Comment (Nastya-Korsak) [reply Module_2/Task_1/HelloLibrary/Class1.cs]: added handling of null and string.Empty cases

File: Module_2/Task_1/WinForms/Form1.cs (hunk 4 above)
Comment (Nastya-Korsak) [reply Module_2/Task_1/WinForms/Form1.cs:2]: added handling for null and string.Empty cases

Comment (Nastya-Korsak) [reply Module_2/Task_1/WinForms/Form1.cs:2]: Created WPF framework

**Pull request merged**